FirebaseRemoteConfig/Sources/RCNConfigRealtime.m

```diff
@@ -58,6 +58,7 @@
 static NSInteger gMaxRetryCount;
 static NSInteger gRetrySeconds;
 static bool gIsRetrying;
```

Comment:
I think these parameters are safer to be set as ivar and set/get it with a function guarded by a serial queue.

Reply:
moved to ivar and wrapped with serial queue